packages/frontend/src/utils/wallet.ts

```diff
@@ -1239,7 +1279,7 @@ export default class Wallet {
             finality: 'final',
         });
         const blockNumber = block.header.height.toString();
-        const signer = account.inMemorySigner || account.connection.signer;
+        const signer = account.signerIgnoringLedger || account.connection.signer;
```

Comment:
would `account.signerIgnoringLedger` ever be defined? I don't see any extended classes that implement that method on `class Account`

Actually... looking at how this method is used, the passed in variable isn't actually of type `Account`- its of type `Wallet`. So in that case, the signer would be defined. Maybe we should change the variable name? Its kinda confusing overall though.

Reply:
not only this thing need to be rewrite, in fact the whole file need to rewrite.

But first, let's just merge this file into master first, in order to avoid more conflict in the future